code/datalab-app/src/api/schema/index.js

```diff
@@ -0,0 +1,45 @@
+import {
+  GraphQLSchema,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLInt,
+} from 'graphql';
+
+let count = 0;
+
+const RootQueryType = new GraphQLObjectType({
```

Comment:
I don't know where it goes, but we want all GraphQL entries to have a description so it appears in the GraphiQL UI.